### DISCUSSION TITLE
Welcome to NETworkManager Discussions!

### DISCUSSION THREAD

**Discussion by BornToBeRoot** [BornToBeRoot/NETworkManager#453] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by chenjlin** [comment]
There is no Chinese to choose from the installation package

**Reply by BornToBeRoot** [reply]
Chinese was added in the latest release :)

**Reply by itholl** [reply]
Unable to set the language, select the target language, restart the application does not take effect

**Reply by BornToBeRoot** [reply]
Which language?

**Reply by yuwolf21** [reply]
china

**Comment by HarveyLi2019** [comment]
Yes, how to set Chinese

**Reply by BornToBeRoot** [reply]
https://github.com/BornToBeRoot/NETworkManager/issues/1281

**Comment by BornToBeRoot** [comment]
@HarveyLi2019 @itholl @yuwolf21 @chenjlin 

The current release includes the chinese language files https://github.com/BornToBeRoot/NETworkManager/releases/tag/2021.11.30.0
